components/suggestion/at_mention_provider/at_mention_provider.tsx

```diff
@@ -271,11 +297,17 @@ export default class AtMentionProvider extends Provider {
             filter((member) => !localUserIds[member.id]).
             sort(orderUsers);
 
-        return priorityProfiles.concat(localAndRemoteMembers).concat(localAndRemoteGroups).concat(specialMentions).concat(remoteNonMembers);
+        return [
+            ...priorityProfiles,
+            ...localAndRemoteMembers,
+            ...localAndRemoteGroups,
+            ...specialMentions,
+            ...remoteNonMembers,
+        ].map((item: (LocalMember | LocalGroup | SpecialMention)) => item as Item);
     }
 
     // updateMatches invokes the resultCallback with the metadata for rendering at mentions
-    updateMatches(resultCallback, items) {
+    updateMatches(resultCallback: (params: ResultCallbackParams) => void, items: any[]): void {
```

Comment:
Should this `any` be type `Item`?

```suggestion
    updateMatches(resultCallback: (params: ResultCallbackParams) => void, items: Item[]): void {
```

Reply:
Fixed